protos/in_toto_attestation/v1/statement.proto

```diff
@@ -15,12 +16,7 @@ message Statement {
   // Expected to always be "https://in-toto.io/Statement/v1"
   string type = 1 [json_name = "_type"];
 
-  message Subject {
-    string name = 1;
-    map<string, string> digest = 2;
-  }
-
-  repeated Subject subject = 2;
+  repeated in_toto_attestation.v1.ResourceDescriptor subject = 2;
```

Comment:
We cannot set expectations here about the required `digest` field. Validation has to occur separately, I'm again thinking of something like https://www.kusari.dev/blog/spector-announcement/.

Reply:
TBF the old definition wasn't setting that as mandatory either.